pkgs/applications/audio/hydrogen/default.nix

```diff
@@ -27,7 +27,7 @@ stdenv.mkDerivation rec {
   meta = with lib; {
     description = "Advanced drum machine";
     homepage = "http://www.hydrogen-music.org";
-    license = licenses.gpl2Only;
+    license = licenses.gpl2Plus;
```

Comment:
Is it gplPlus or gpl2Plus? You wrote contradicting things in the pr message.

Reply:
Arf. I fixed the pr message.